src/sql/platform/bootstrap/browser/bootstrapService.ts

```diff
@@ -38,26 +39,40 @@ function createUniqueSelector(selector: string): string {
 
 let platform: PlatformRef;
 
-export function bootstrapAngular<T>(service: IInstantiationService, moduleType: IModuleFactory<T>, container: HTMLElement, selectorString: string, params: IBootstrapParams, input?: IEditorInput, callbackSetModule?: (value: NgModuleRef<T>) => void): string {
-	// Create the uniqueSelectorString
-	let uniqueSelectorString = createUniqueSelector(selectorString);
-	let selector = document.createElement(uniqueSelectorString);
-	container.appendChild(selector);
+export function bootstrapAngular<T>(service: IInstantiationService, moduleType: IModuleFactory<T>, container: HTMLElement, selectorString: string, params: IBootstrapParams, input?: IEditorInput, callbackSetModule?: (value: NgModuleRef<T>) => void): Promise<string> {
+	const doBootstrap = ((platform: PlatformRef): string => {
+		// Create the uniqueSelectorString
+		let uniqueSelectorString = createUniqueSelector(selectorString);
+		let selector = document.createElement(uniqueSelectorString);
+		container.appendChild(selector);
+		platform.bootstrapModule(moduleType(params, uniqueSelectorString, service)).then(moduleRef => {
+			if (input) {
+				input.onDispose(() => {
+					moduleRef.destroy();
+				});
+			}
+			if (callbackSetModule) {
+				callbackSetModule(moduleRef);
+			}
+		});
 
-	if (!platform) {
-		platform = platformBrowserDynamic();
-	}
+		return uniqueSelectorString;
+	});
+
+	return new Promise((resolve) => {
+		// First time this function is being called
+		if (!platform) {
+			service.invokeFunction((accessor) => {
+				const environmentService = accessor.get(IEnvironmentService);
+				if (environmentService.isBuilt) {
```

Comment:
My point is, if the product is built you don't want to enable prod mode. I could be wrong but is built is true if the product was built locally as opposed to a production build. You should test though.

Reply:
nvm i checked im wrong.